### PR TITLE
New request

### PR DESCRIPTION
**Pull request by surendram1234**

### CODE REVIEW

File: dev/components/home/HomePage.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 class HomePage extends React.Component {
     render() {
         return (<div>
-            Home Page
+            Home Page same33233
```
Comment (singhsheetal) [inline dev/components/home/HomePage.js]: change 33 to 22

File: dev/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { render } from 'react-dom';
 import App from './components/app';
 
 var HelloWorld = (name) => {
-    return "Hello" + name;
+    return "Hello sds121" + name;
```
Comment (singhsheetal) [inline dev/index.js]: change 121 to 222

**Review by singhsheetal — COMMENTED** [review]
Please fix the changes

**Comment by surendram1234** [comment]
Take changes

**Pull request closed without merging**